packages/mockserver-config-writer/src/prompt/index.ts

```diff
@@ -0,0 +1,32 @@
+import { join } from 'path';
+import { create as createStorage } from 'mem-fs';
+import { create } from 'mem-fs-editor';
+import type { Editor } from 'mem-fs-editor';
+import type { PromptObject } from 'prompts';
+import type { Manifest } from '@sap-ux/project-access';
+import { getMainServiceDataSource } from '../app-info';
+
+/**
+ * Return the list of questions to configure mockserver.
+ *
+ * @param params - optional parameters used to fill default values
+ * @param params.webappPath - optional path to webapp folder, where manifest is
+ * @param params.fs - optional memfs editor instance
+ * @returns - array of questions that serves as input for prompt module
+ */
+export function getMockserverConfigQuestions(params?: { webappPath?: string; fs?: Editor }): PromptObject[] {
+    let path = '';
+    if (params?.webappPath) {
+        const fs = params.fs || create(createStorage());
+        const manifest: Manifest = JSON.parse(fs.read(join(params.webappPath, 'manifest.json')));
+        path = getMainServiceDataSource(manifest)?.uri || '';
```

Comment:
Why do you limit it to the main service? Wouldn't this also be useful for an `OVP` app that might have multiple services?

Reply:
This is the initial default value when prompting for the path to mock, but you are right, when we have an OVP or multiple OData services as data sources, we should allow to select from those. I've changed this now.